src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/StringCollectionTextWriter.cs

```diff
@@ -28,7 +29,7 @@ public class StringCollectionTextWriter : TextWriter
         public StringCollectionTextWriter(Encoding encoding)
         {
             _encoding = encoding;
-            Buffer = new BufferEntryCollection();
+            Content = new BufferedHtmlContent();
```

Comment:
This is changing because we removed the `StringCollectionTextWriter` constructor in `HtmlString` which is used here - https://github.com/aspnet/Mvc/pull/2797/files#diff-0e5ef27109c7145789f12182064f84acL423